scripts/install_new.sh

```
@@ -81,6 +81,10 @@ install_from_run() {
     fi
 }
 
+ONEC_MAJOR_VER=$(echo "$ONEC_VERSION" | cut -d'.' -f1,2 | tr -d '.')
+
+cd /tmp/downloads/Platform${ONEC_MAJOR_VER}/${ONEC_VERSION}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)